python/hyperspectral/spectra/spectra.py

```diff
@@ -0,0 +1,203 @@
+from collections import Counter
+from copy import deepcopy
+from functools import reduce
+
+import numpy as np
+import parsec
+import scipy.interpolate as interpolate
+
+class Spectrum:
```

Comment:
Could we add a short docstring here describing what this class represents?

Reply:
Would it be possible also to add some primitive example of its usage? 
Anything - i.e. read some file & apply the plot function.